src/schema/v2/schema.ts

```diff
@@ -95,6 +96,7 @@ const rootFields = {
   // articles: Articles,
   artwork: Artwork,
   // artworkVersion: ArtworkVersionResolver,
+  artworksByInternalID: ArtworksByInternalID,
```

Comment:
interesting! how come the more specific name, vs just `artworks` (where one of the arguments, as of now, is a list of internal id's)? I don't have a strong opinion one way or the other, just wondering!

Reply:
We want to encourage use of the more powerful `artworksConnection` right? Wanted to avoid adding a top-level `artworks` so folks don't expand it at all.